test/conf/cassandra.yaml

```diff
@@ -53,6 +53,7 @@ materialized_views_enabled: true
 drop_compact_storage_enabled: true
 file_cache_enabled: true
 auto_hints_cleanup_enabled: true
+accord_transactions_enabled: true
```

Comment:
don't you need to change this config for JVM dtest as well?

Reply:
I changed it in `AccordTestBase`, of course, but outside that, my goal was to keep the rest of the in-JVM test landscape running w/ the default, which right now means `accord_transactions_enabled: false`. Right trade-off, or am I thinking about it wrong?

Reply:
You know what...I think you're right. It's probably better to have all of this active in a testing context so we ferret out problems earlier. Changing...